proto/btc/sign_txn.proto

```diff
@@ -51,7 +51,7 @@ enum SignTxnStatus {
 
 message SignTxnInitiateRequest {
   bytes walletId = 1;
-  repeated common.DerivationPath derivationPath = 2;
+  repeated uint32 derivationPath = 2;
```

Comment:
Preferably also remove common.DerivationPath struct.

Reply:
missed it. thanks